src/bijectors/corr.jl

```diff
@@ -297,11 +297,10 @@ function _link_chol_lkj(W::AbstractMatrix)
     # Some zero filling can be avoided. Though diagnoal is still needed to be filled with zero.
 
     @inbounds for j in 1:K
-        remainder_sq = one(eltype(W))
         for i in 1:(j - 1)
```

Comment:
You could save a few operations by reversing the loop and summing `remainder_sq` incrementally.